.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+name: Beagle Release
+
+on:
+  workflow_run:
+    workflows: [ "Beagle Yoga Release" ]
+    types:
+      - completed
```

Is the idea to generate manually or by generating a tag?
```suggestion
  push:
    tags:
      - '*'
```